build-config/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -205,6 +205,11 @@
       <property name="message" value="Properties should either be declared 'validate = &quot;notNull&quot;' or 'get = &quot;optional&quot;'"/>
       <property name="illegalPattern" value="true"/>
     </module>
+    <module name="Regexp">
+      <property name="format" value="\{[ ]*(\n|\r\n)[ ]*(\n|\r\n)[ ]*}"/>
+      <property name="message" value="Empty blocks should not contain empty lines"/>
```

Comment:
This will blow up on marker interfaces

Reply:
You mean like 
```
interface IsAFoo {

}
```
?

Reply:
Yeah that

Reply:
Is the rule no blank lines in empty blocks or just no blank lines in empty constructors? The former seems to make more sense (or at least, the former seems to be more consistent).

Reply:
Where are you getting this rule from?

Reply:
In my mind I don't really care about a single blank line in a constructor, although no blank lines might be the way I would write my own.
Otherwise I would actively enforce a single blank line on a marker interface or class definition.
Perhaps we should have a rule saying multiple sequential blank lines are disallowed?

Reply:
> In my mind I don't really care about a single blank line in a constructor

Your review comments to the contrary are the whole reason behind this PR. I'll close it and just link you back to this next time 😄